roles/openshift_autoheal/tasks/install.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Check that the AWX address has been provided
+  fail:
+    msg: "Variable 'openshift_autoheal_awx_address' is mandatory but it isn't defined"
+  when: openshift_autoheal_awx_address is not defined
+
+- name: Check that the AWX password has been provided
+  fail:
+    msg: "Variable 'openshift_autoheal_awx_password' is mandatory but it isn't defined"
+  when: openshift_autoheal_awx_password is not defined
+
+- name: Create a temporary directory for doing work in the target host
+  command: mktemp -td openshift-autoheal-XXXXXX
```

Comment:
didn't you say `-t` is deprecated, and `-p` replaces it?

Reply:
Yes, I did, but I wanted to do what you asked :-) . I can replace it with `-p`, no problem.

Reply:
I don't mind which flag you use as long as it does the job :)